docs/versioning/how-to-version.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+nav_order: 4
+title: How to create versions  
+has_children: false
+parent: Versioning
+---
+# How to create versions in the Registry
+
+## Step 1. Navigate to the item record in the Registry
+
+- Ensure you're logged in.
+- Navigate to the item record in the Registry
+- Click on the "VERSIONING" tab as shown below
+
+|                         Versioning Tab                                                                 |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-versioning-tab.jpg" alt="drawing" width="600"/> |
+
+
+
+## Step 2. Create a new version
+
+- Click on the "Create New Version" button to create a new version of the item.
+
+|               Click on the "Create new version" button                                                      |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-version.png" alt="drawing" width="600"/> |
+
+
+
+## Step 3. Add a reason for the new version
+
+
+|               Add details to the versioning action                                                      |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-reason.png" alt="drawing" width="600"/> |
+
+Upon clicking the "Submit" button, the new version of the item will be created. 
+A set of jobs will be sent to be processed in order for the provenance graphs to be created and updated.
+
+The page will be refreshed to the version of the item you just created.
+
+## Step 5. View the version info
+
+You now have a new version of the item. You can view the version history including the current and previous versions.
+
+|               Updated versioning details                                                    |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-version-result.png" alt="drawing" width="600"/> |
+
+You can click on the icon with 3 dots in the versioning timeline to expand to display additional versions.
```

Comment:
Please highlight this in the above screenshot

Reply:
done